### PR TITLE
Load env vars for blitz new command - ignore cached variables

### PR DESCRIPTION
**Pull request by beerose**
Closes: https://github.com/blitz-js/blitz/issues/3042

### What are the changes and their implications?

Not sure if that would be the best way to handle that (I might have missed something), so would love some suggestions if it's not. 

## Bug Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `canary` branch](https://github.com/blitz-js/blitzjs.com/tree/canary))

### CODE REVIEW

File: nextjs/packages/next-env/index.ts [hunk 1]
```diff
@@ -67,15 +67,16 @@ export function processEnv(
 
 export function loadEnvConfig(
   dir: string = process.cwd(),
-  dev?: boolean,
-  log: Log = console
+  log: Log = console,
+  ignoreCache = false
```
Comment (flybayer) [inline nextjs/packages/next-env/index.ts:71]: Why did you remove `dev`?

Comment (flybayer) [reply nextjs/packages/next-env/index.ts:71]: So `ignoreCache` is what fixes the issue right? Makes sense.

It would be slightly better to add an object as the new param as `{ignoreCache}`. I always fine unnamed function params error prone, especially when we are in a fork situation.

Comment (beerose) [reply nextjs/packages/next-env/index.ts:71]: > Why did you remove dev?

Not used in the function body

Comment (flybayer) [reply nextjs/packages/next-env/index.ts:71]: > Not used in the function body

Normally I would agree, but in the context of a fork, this makes upstream merges more difficult
